docs/home/whats-new-2-16.md

```diff
@@ -69,3 +69,14 @@ date: 2023-Dec-4
 #### OpenShift Support
 
 * Updated installation prerequisites. For more information, see [Kubernetes support matrix](../admin/runai-setup/cluster-setup/cluster-prerequisites.md#releases). -->
+
+## Upcoming Changes
+
+### Project migration
+
+* In a future release, Run:ai will be deprecating the migration of projects between departments. This affects:
+  
+    * API&mdash;the `departmentId` field will be marked as deprecated in the `projects` endpoint.
```

Comment:
...will be marked deprecated in the 'PUT /projects' endpoint

Reply:
@lavianalon only the `put`? what about other endpoints in the projects category? Because I see that `post` has it as well. 
```suggestion
    * API&mdash;the `departmentId` field will be marked as deprecated in both `put` and `post` endpoints in the `projects` category.
```